javers-core/src/main/java/org/javers/core/metamodel/type/TypeMapper.java

```diff
@@ -77,6 +81,13 @@ private void registerCoreTypes(){
         if (ReflectionUtil.isJava8runtime()){
             addType(new OptionalType());
         }
+
+        if (ReflectionUtil.isClassPresent("com.google.common.collect.Multiset")){
```

Comment:
imo, jeśli dodałeś explicite importy na `import org.javers.guava.multimap.MultimapType`
to poleci NoClassDefFound dla userów, którzy nie używają Guava, te importy powinny być schowane w MultisetType. tak jak jest to zrobione z OptionalType